### PR TITLE
Test6

### PR DESCRIPTION
**Pull request by ahmedyasserarafat**
Started

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "rulesDirectory": [
-    "node_modules/codelyzer123"
+    "node_modules/codelyzer1093"
```
Comment (arafatahmedyasser) [inline tslint.json:3]: What is This

**Review by arafatahmedyasser — COMMENTED** [review]
Pls provide some valid value

File: tslint.json (hunk 1 above)
Comment (ahmedyasserarafat) [reply tslint.json:3]: Latest Value

File: Important.txt [hunk 2]
```diff
@@ -1,3 +1,2 @@
 DER---
-FTT
```
Comment (arafatahmedyasser) [inline Important.txt:2]: 1

File: mysupply.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
 One,Two,Three,Four
-Five,six,seven,eight,
```
Comment (arafatahmedyasser) [inline mysupply.txt:2]: Two

**Review by arafatahmedyasser — APPROVED** [review]

File: tslint.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "rulesDirectory": [
-    "node_modules/codelyzer123"
```
Comment (arafatahmedyasser) [inline tslint.json:3]: HUI

**Review by arafatahmedyasser — CHANGES_REQUESTED** [review]

File: mysupply.txt (hunk 3 above)
Comment (arafatahmedyasser) [inline mysupply.txt:2]: Help thus

**Review by arafatahmedyasser — CHANGES_REQUESTED** [review]
Hop

File: mysupply.txt (hunk 3 above)
Comment (ahmedyasserarafat) [reply mysupply.txt:2]: Ok

**Pull request merged**